File: 2026POTD/JANUARY/25-01-2026/1984.dart
class Solution {
  int minimumDifference(List<int> nums, int k) {
    if (k <= 1) return 0;
    nums.sort();
    int ans = 1 << 30;
    for (int i = 0; i + k - 1 < nums.length; i++) {
      ans = ans < nums[i + k - 1] - nums[i]
          ? ans
          : nums[i + k - 1] - nums[i];
    }
    return ans;
  }
}

File: 2026POTD/JANUARY/25-01-2026/1984.php
class Solution {
    function minimumDifference($nums, $k) {
        if ($k <= 1) return 0;
        sort($nums);
        $ans = PHP_INT_MAX;
        $n = count($nums);
        for ($i = 0; $i + $k - 1 < $n; $i++) {
            $ans = min($ans, $nums[$i + $k - 1] - $nums[$i]);
        }
        return $ans;
    }
}